.env

```diff
@@ -13,9 +13,19 @@ See the License for the specific language governing permissions and
 limitations under the License.
 =========================================================================-->
 
-ISOLATION_ID=latest
+# ISOLATION_ID=latest
```

Comment:
Is there a reason to keep ISOLATION_ID now that it's commented out? I don't see it assigned/appended anywhere, just disabled. Are there still references to it elsewhere in the codebase?

Reply:
It is currently being referenced in docker yaml files by the image value for certain services.  

```
image: rbac-server-production:${ISOLATION_ID-latest}
````